src/service/performance-impl.js

```diff
@@ -97,7 +97,7 @@ export class Performance {
         });
 
     // Tick window.onload event.
-    loadPromise(win).then(() => {
+    loadPromise(win.document).then(() => {
```

Comment:
going to ping @cramforce on this one as this changes what we're measuring. domready is going to fire much earlier than window.onload